packages/fxa-settings/src/pages/CompleteResetPassword/index.test.tsx

```diff
@@ -0,0 +1,117 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { mockAppContext, renderWithRouter } from '../../models/mocks';
+import { AppContext, AlertBarInfo } from '../../models';
+import CompleteResetPassword from '.';
+import { usePageViewEvent } from '../../lib/metrics';
+
+jest.mock('../../lib/metrics', () => ({
+  logViewEvent: jest.fn(),
+  usePageViewEvent: jest.fn(),
+}));
+
+const mockNavigate = jest.fn();
+jest.mock('@reach/router', () => ({
+  ...jest.requireActual('@reach/router'),
+  useNavigate: () => mockNavigate,
+}));
+
+const alertBarInfo = {
+  success: jest.fn(),
+  error: jest.fn(),
+} as unknown as AlertBarInfo;
+
+describe('CompleteResetPassword', () => {
+  it('renders the component as expected when the link is valid', () => {
+    renderWithRouter(
+      <AppContext.Provider value={mockAppContext({ alertBarInfo })}>
+        <CompleteResetPassword />
+      </AppContext.Provider>
+    );
+
+    const createNewPasswordHeader = screen.getByRole('heading', {
+      name: 'Create new password',
+    });
+    const passwordRequirements = screen.getByText('Password requirements');
```

Comment:
FWIW you don't have to set `getBy`s to constants and check that they're in the doc since `screen.getByText('Password requirements');` runs the test, but I know Mill preferred explicitly testing for them like you have here, so 🤷‍♀️

Reply:
To clarify, are you suggesting to 
1. remove the `expect(...).toBeInTheDocument()` and only keep the `getBy` 
(e.g., test with only `screen.getByText('Password requirements');`)
OR
2. nest the `getBy` in the expect statement 
(e.g., `expect(screen.getByText('Password requirements').toBeInTheDocument()`)?

Reply:
I personally prefer `screen.getByText('Password requirements');` because that tests if that text is on the screen. We're not really consistent with that though so it was more of a side note / FYI.